dev-support/hbase_docker/m1/Dockerfile

```diff
@@ -0,0 +1,92 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM --platform=linux/amd64 ubuntu:22.04 AS base_image
```

Comment:
`--platform=linux/amd64` should be the only change, so it looks good to me.

Reply:
I don't think this is the right way to do things. Instead, you should provide `--platform` to the docker command line invocation when building the image. That will cause it to pull down the ubuntu image of the specified platform.

I left a comment on the Jira with an example of how to do a multi-platform docker image. It's not the only way to do it, but it's the style I've been using in the kustomize repo. Yetus project may have an alternative example of how to achieve this.